vendor/bat-native-ledger/src/DEPS

```diff
@@ -0,0 +1,3 @@
+include_rules = [
+ # "+rapidjson",
```

Comment:
nit: commented out DEP, is this still required?

Reply:
no, it was moved further down and I forgot to remove it

Reply:
I'll remove after the CI run finishes to I don't interrupt it